cpp/benchmarks/CMakeLists.txt

```diff
@@ -183,6 +183,11 @@ ConfigureNVBench(
   sort/sort_lists.cpp sort/sort_structs.cpp
 )
 
+# ##################################################################################################
+# * create benchmark
```

Comment:
Wait, I missed this. Please update comment here. Also, the new line below should be moved closer to the other struct benchmarks.

Reply:
Thanks for pointing it out, I missed it too.

Reply:
Other struct related benchmarks are in different sub directories eg: sort, groupby

Reply:
Resolving this conversation. Please let me know if other any changes are required, thanks!